tutorials/snake/steps/step-3.md

```diff
@@ -60,7 +60,7 @@ We will render the grid using a [`(GUI.Table)`](https://docs.fragcolor.xyz/block
 
 ## Populating the grid
 
-Before we can draw anything, we need to update the grid with the fruit and the snake. To update a sequence at a given index, we can use the [`(Assoc)`](https://docs.fragcolor.xyz/blocks/General/Assoc/) block. And since the snake is saved at a sequence itself, we need to iterate through all its elements. However, the head, tail and body are represented by different value, so we will handle them separately.
+Before we can draw anything we need to update the grid with the fruit and the snake. To update a sequence at a given index, we can use the <a href="https://docs.fragcolor.xyz/blocks/General/Assoc/" target="_blank">`Assoc`</a> ) block. And since the snake is saved as a sequence itself, we need to iterate through all its elements. However, the head, tail and body are represented by different values, so we will handle them separately.
```

Comment:
No HTML

Reply:
Reverted to markdown link tag in latest commit.